javaagent-instrumentation-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/internal/InstrumentedTaskClasses.java

```diff
@@ -38,18 +38,18 @@ protected Boolean computeValue(Class<?> taskClass) {
         }
       };
 
-  private static volatile Trie<Boolean> ignoredTaskClasses;
+  private static volatile Predicate<String> ignoredTaskClassesPredicate;
 
   /**
-   * Sets the configured ignored tasks trie. This method is called internally from the agent
-   * classloader.
+   * Sets the configured ignored tasks predicate (that uses a trie to find the common prefix). This
```

Comment:
Why is it important to mention trie here?

Reply:
It's not, now that I think of it. Removed 👍